src/pywrangler/utils.py

```diff
@@ -3,8 +3,14 @@
 from pathlib import Path
 
 import click
+from functools import cache
 from rich.logging import Console, RichHandler
 from rich.theme import Theme
+from .metadata import PYTHON_COMPAT_VERSIONS
+from datetime import datetime
+from typing import Literal
+import pyjson5
+import tomllib
```

Comment:
This drops support for Python <3.11, but I guess that's fine?

Reply:
We didn't declare a dependency on tomli in pyproject.toml so it already didn't work.